pom.xml

```diff
@@ -376,6 +377,28 @@
         </executions>
       </plugin>
 
+      <!-- Enforce versions -->
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-enforcer-plugin</artifactId>
+        <version>1.4</version>
+        <executions>
+          <execution>
+            <id>enforce-versions</id>
+            <goals>
+              <goal>enforce</goal>
+            </goals>
+            <configuration>
+              <rules>
+                <requireJavaVersion>
```

Comment:
emm. the problem is we only want to enforce hadoop version (>=2.4.0) in pom of  `integration-yarn` module, not the parent pom. If yarn module is not activated, hadoop version could be 1.0.4 or later